kura/container/README.md

```diff
@@ -53,7 +53,7 @@ The JMX port defined is 9010 and the Remote debug port is 9011. Both ports are n
 
 ## Re-Building
```

Comment:
Can you update the docs to show the correct endpoint for the GUI? Right now it points to `localhost:8080` but is should be `localhost:8443` if the container is run using the command suggested in the docs.

I'm referring to the line 9 of the README:

> Once the image is started you can navigate your browser to http://localhost:8080 and log in using the credentials `admin` : `admin`.